src/editor/js/editor/index.js

```diff
@@ -0,0 +1,102 @@
+export class Editor {
+    static ROWS_COUNT_DEFAULT = 16;
+    static COLS_COUNT_DEFAULT = 16;
+    static COLOUR_DEFAULT = '#000000';
+    static CELL_WIDTH_DEFAULT = 20;
+    
+    canvas;
+    
+    #isDrawing = false;
+    #colour = Editor.COLOUR_DEFAULT;
+    #width = Editor.CELL_WIDTH_DEFAULT;
+    
+    constructor() {
+        this.build();
+    }
+    
+    /**
+     * Build cells
+     * @param rows {number} - count of rows
+     * @param cols {number} - count of cols
+     *
+     * @return void
+     */
+    build(
+        rows = Editor.ROWS_COUNT_DEFAULT,
+        cols = Editor.COLS_COUNT_DEFAULT
+    ) {
+        this.canvas = document.createElement('div');
+        this.canvas.classList.add('canvas');
+        
+        for (let r = 0; r < rows; r++) {
+            const row = document.createElement('div');
+            row.classList.add('canvas__row');
```

Comment:
I would recommend adding id here.

OR! You can store link to element in the data grid along with the cell color. Looks like good solution  to me ;)